### ISSUE TITLE
Can't add Hue Plugs

### ISSUE THREAD

**Issue by misterwesley** [Gh61/lovelace-hue-like-light-card#13]
When I try to add Hue Plugs or on/off light switches it does not appear in the Hue Screen.
Can this be added?

**Comment by Gh61** [comment]
I don't have these type of devices, so I can't test it directly. But it should be working without problems.
Please post your card settings and entity attributes/info.

When you open your HA, go to Developer Tools, then States and search for lights you have problem with. 
Click on it to open detail of the entity and make me a printscreen of attributes.

**Comment by misterwesley** [comment]
Card settings:
`type: custom:hue-like-light-card
title: Schuur
icon: mdi:greenhouse
onClickAction: hue-screen
offClickAction: hue-screen
onClickData: hue-screen
offClickData: hue-screen
entities:
  - light.schuur
  - switch.kantoor_plug`

![Hue Plug](https://user-images.githubusercontent.com/121464535/222991396-5ee57c27-7ca2-4879-952e-11c7af45b069.jpg)
![Other Light](https://user-images.githubusercontent.com/121464535/222991513-c685736e-29f3-4014-9905-5eb6543e8163.jpg)

**Comment by misterwesley** [comment]
When I don't have it in an vertical stack I get the error "this._ctrl is undefined"

**Comment by Gh61** [comment]
The light.schuur should work fine, but the switch domain is currently not supported, as it is not `light`.
There should even be an error.

**Comment by misterwesley** [comment]
I made a helper that switches the type of switch.kantoor_plug to a light and now it works.
I already had that for light.schuur which is also a switch but I did nothing and that one also works now.

**Comment by AgileCowboy** [comment]
Thanks for this solution @Gh61 and @misterwesley. Being relatively new to Home Assistant I was struggling with a user friendly solution which Hue Like Light cards provides. I also struggled with adding the Philips Hue Smart plugs to the interface. 

To add a bit of detail to your solution @misterwesley, I've used the build-in [Switch as X (Change device type of a switch)](https://www.home-assistant.io/integrations/switch_as_x/) integration to change the "switch." prefix of my PH Smart Plug to "light.". With this entity type the plug is now shown in the Hue Like card and can be operated as required and expected.
